### PR TITLE
Update App Cards with Server Running Information

### PR DESCRIPTION
**Pull request by jbouder**
## Reference Issues or PRs

- N/A

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [x] Did you add new tests?

## Screenshots
Screenshot 2024-05-28 at 12 04 13 PM

## Any other comments?
- Updates to display server info and stop button on running apps
- Refactor to simplify app card updates in the future

### CODE REVIEW

File: ui/src/pages/home/apps-section/app-grid/app-grid.tsx [hunk 1]
```diff
@@ -11,16 +11,11 @@ export const AppGrid = ({ apps }: AppsGridProps): React.ReactElement => {
...
           title={app.name}
-          description={app.description}
-          thumbnail={app.thumbnail}
-          framework={app.framework}
-          url={app.url}
           serverStatus={app.status}
           lastModified={app.last_activity}
-          username={app.username}
           isPublic={app.public}
           isShared={app.shared}
```
Comment (kildre) [inline ui/src/pages/home/apps-section/app-grid/app-grid.tsx:20]: Could these props be renamed to simplify even further, i.e. title becomes name, serverStatus changed to status, isPublic to public

Comment (jbouder) [reply ui/src/pages/home/apps-section/app-grid/app-grid.tsx:20]: Some could be, but 1 in particular cannot, `public`, as it has meaning in TypeScript.

Comment (jbouder) [reply ui/src/pages/home/apps-section/app-grid/app-grid.tsx:20]: And I try to stick with camel casing for variables and props, whereas some of the values coming from the API are snake cased.

Comment (kildre) [reply ui/src/pages/home/apps-section/app-grid/app-grid.tsx:20]: Ok. Based on that. Closing this.

**Review by kildre — APPROVED** [review]
Looks good.

**Pull request merged**